Editor/Strings.cs

```diff
@@ -136,5 +136,48 @@ public static class Strings
         public const string TabAnywhere = "TabAnywhere";
         public const string TabManagedEC2 = "TabManagedEC2";
         public const string TabHelp = "TabHelp";
+
```

Comment:
For new strings, we should include the context of the string in the naming convention. Check out [PR 93's Strings changes](https://github.com/aws/amazon-gamelift-plugin-unity/pull/94/files#diff-95908034dec0ae86781054b3193a4281f64aadabe0857e81f5cfc8d0a0392130) for an example